iOS/Flows/Conversation/Cells/MessageCell.swift

```diff
@@ -123,6 +123,49 @@ class MessageCell: UICollectionViewCell {
         self.handleDetailVisibility(areDetailsFullyVisible: areDetailsFullyVisible)
     }
 
+    private var messageController: MessageController?
+    private var messageSubscriptions: Set<AnyCancellable> = []
+    private var messageTasks = TaskPool()
```

Comment:
I moved message subscription out of the footer view and into the parent message cell. This gives us a little more control over when GetMessage is called.